shared/constants/types/flux.js

```diff
@@ -20,3 +24,41 @@ export type Dispatch = (action: AsyncAction | Action) => ?Promise<*>
 
 export type TypedAsyncAction<A> = (dispatch: TypedDispatch<A>, getState: GetState) => ?Promise<*>
 export type TypedDispatch<-A> = (action: TypedAsyncAction<A> | A) => ?Promise<*>
+
+function _deepKeysOnly (m: Object) {
+  return Object.keys(m).reduce((acc, k) => {
+    if (typeof m[k] === 'object') {
+      acc[k] = _deepKeysOnly(m[k])
+    } else {
+      acc[k] = null
+    }
+    return acc
+  }, {})
+}
+
+function _shallowKeysOnly (m: Object) {
```

Comment:
What is the expected output of this function? Looks like it'll just return an object with keys all set to null. Is that intended?